File: chapter_2/hello.py
NUMBER = 5

print("Code in a loop:")
i = 0
while i < NUMBER:
    print(i, "Hello World")
    i += 1

print("Code in a function:")
def hello(i=0):
    print(i, "Hello World")
    i += 1
    if i >= NUMBER:
        return
    else:
        hello(i)
hello()

File: chapter_2/hello.js
let NUMBER = 5;

document.write("Code in a loop:<br />")
let i = 0;
while (i < NUMBER) {
	document.write(i++ + ", Hello world.<br />");
}

document.write("Code in a loop:<br />")
function hello(i) {
	if (i === undefined) {
		i = 0;
	}
	document.write(i++ + ", Hello world.<br />");
	if (i >= NUMBER) {
		return;
	} else {
		hello(i);
	}
}
hello();
